contributing/ansible-development.rst

```diff
@@ -0,0 +1,99 @@
+Ansible roles development
+=========================
+
+.. _Ansible: https://www.ansible.com/
+
+This document describes the conventions and process used by the OME team for
+developing, maintaining and releasing its Ansible_ roles.
+
+The set of rules and procedures described below applies to the official
+OME roles registered in https://github.com/ome/ansible-roles.
+
+Source code
+-----------
+
+The source code of a Java library should be maintained under version control
+using Git_ and hosted on GitHub_ under the
+`openmicroscopy <http://github.com/openmicroscopy/>`__ organization.
+Git repositories should be named as `ansible-role-<ROLE_NAME>`. Each directory
```

Comment:
Given past problems worth mentioning something about hyphens and `galaxy_info.role_name`, either here or in the `Distribution and support` section since it relates to `meta/main.yml`?